apps/docs/src/app/examples/inputs/DropdownSuffix.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import { HvInput, HvSelect } from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  const [email, setEmail] = useState("");
+  const [role, setRole] = useState<string | null>(null);
+
+  return (
+    <HvInput
+      label="Dropdown suffix"
+      className="w-300px"
+      onChange={(_, value) => {
+        setEmail(value);
+      }}
+      value={email}
+      placeholder="Type user email..."
+      endAdornment={
+        <HvSelect
+          enablePortal
+          value={role}
+          onChange={(_, val) => setRole(val)}
+          classes={{
+            root: "w-100px! border-l-1! border-l-border! rounded-none!",
```

Comment:
<img width="153" height="79" alt="Image" src="https://github.com/user-attachments/assets/bd8d2527-6b9e-4294-a7a2-a18cf5306c5e" />

maybe let's give this some flexibility?
a `defaultValue` would also be nice (or a `placeholder` instead) - the empty Select is strange

```suggestion
            root: "max-w-140px border-l-1! border-l-border! rounded-none!",
```